spec/models/mixins/miq_policy_mixin_spec.rb

```diff
@@ -0,0 +1,72 @@
+describe MiqPolicyMixin do
```

Comment:
I think I would rather see this as a `shared_context`, and then this applied to all of the models that support this mixing.  Specs like this end up becoming invalid if `Host` ever stops supporting this, but then it is still used elsewhere.

Adds more runtime to the suite, but it is a much more comprehensive way of testing this in all models that include it.

Reply:
I'll make `Host` more generic - see if that makes this more clear / gets us closer to the `shared_context` example.

I tend to expect these specs to include the mixin in a generic object. But since these are active record objects, the cost seemed a bit high. Longer term it will probably be better to go with `shared_context`

Reply:
@NickLaMuro I'm so torn on this one. I totally get why you say this.

I'm writing a method for `tag_attribute` - I think in that process, I'll have the infrastructure to make the `shared_context` work easier - hopefully?

Reply:
updated

Reply:
Okay, well, now that I wrote the code to do this, I guess this is good now :trollface: